docs/guides/earning-cred.md

```diff
@@ -1,42 +1,41 @@
+The types of contributions and plugins are likely to increase. But for now, small is beautiful. Your contributions are tracked across the these active plugins in the following ways:
 
 
 | 💬 [Discord] | 🧵 [Discourse] | 🦠 [Github]| 🥇 [Initiatives]
 | -- | -- | -- | -- |
-|someone gives your post : <img width="20" alt="SourceCred" src='https://sourcecred.io/img/favicon.png' /> or :100: or :+1:| someone gives your post/reply a :heart: | You open a pull request  | you make a specific task approved by the community
+|someone gives your post any emoji : <img width="20" alt="SourceCred" src='https://sourcecred.io/img/favicon.png' /> :100: :+1:| someone gives your post/reply a :heart: | You open a pull request  | you accomplish a specific task approved by the community
 
+### 🔥 Radiation
+Since Cred can be created from nothing by community members, there must be a balance so that Cred maintains its worth. Our system is called **Cred Radiation**, and is similar to how water evaporates from all water sources. For every node on the Contribution Graph, a fixed proportion of its Cred radiates away, and that fixed proportion is called alpha.
 
+### 🏄🏾 Cred Flowing
+Every contributor gets a Cred score. Your Cred Score is the amount of Cred that is flowing *through* your pond - the amount entering is the same amount that is simultaneously leaving, flowing to other community members. Each of your contributions is interconnected with other community contributions across different platforms. Contributions are even connected through time - today's contributions connect to contributions from yesterday, last week, etc.
 
+To put it differently, you are part of an environment that sustains itself across time where water is key. Water will flow in different ways. It will flow between contributor ponds (Cred Flowing), fall from the sky as rain (Cred Minting) and evaporate from everyone's ponds over time (Cred Radiation).
 
-### 🏄🏾Cred flowing
-All the contributions you and your community make are on the same Contribution Graph. Each of your contributions is interconnected with other community contributions, across different platforms. Contributions are even also connected through time - today's contributions connect to contributions from yesterday, last week, etc
-
-To put it differently, you are part of an environment that sustains itself across time for which water is key. In your pond, water will not only flow from the sky. Indeed, being connected with other ponds will make the exchange of water possible. It might be drops or a steady stream depending on the size of your connections. And at any point the sky&sun will make sure that some of your drops evaporate to keep everything in balance
-
-
-### 💡 At a conceptual level 
+### 💡 At a Conceptual Level 
 
 When you react to a message with <img width="20" alt="SourceCred" src='https://sourcecred.io/img/favicon.png' /> in one of the plugins :
 
   - it creates a new reaction node in the graph, which mints Cred 
```

Comment:
The <img width="20" alt="SourceCred" src='https://sourcecred.io/img/favicon.png' /> emoji now flows I think 10X more Cred than regular emoji? If we're using Discord as an example (which I think makes sense), we should update to reflect that.

Reply:
It does, but that's only true in our community, not by default.

Reply:
Here's my rewrite so that all of this is clear:

The types of contributions and plugins tracked by the SourceCred algorithm are likely to increase. Currently, your contributions are tracked across the these active plugins in the following ways:


| 💬 [Discord] | 🧵 [Discourse] | 🦠 [Github]| 🥇 [Initiatives]
| -- | -- | -- | -- |
|Someone gives your post any emoji 👍 💯<img width="20" alt="SourceCred" src='https://sourcecred.io/img/favicon.png' />| Someone gives your post/reply a ❤️ | You open a pull request  | You accomplish a specific task approved by the community

 Each community can also change the weights of how much Cred is minted from each action or reaction. For example, the SourceCred community uses the SourceCred emoji to denote extra value, so it mints more Cred than other emojis. If a community values GitHub pull requests but uses Discourse for down time chats, they could make it so Discourse posts/replies mint little Cred and GitHub activity mints more.